node/packages/aws-lambda-otel-extension/external/otel-extension-external/index.js

```diff
@@ -268,7 +273,12 @@ module.exports = (async () => {
               case 'eventData':
                 {
                   if (data.record.requestEventPayload) {
-                    sendReport('request', data.record.requestEventPayload);
+                    const { resourceAtt, metricsAtt } = createAttributes(data.record);
+                    sendReport('request', {
+                      ...data.record.requestEventPayload,
+                      attributes: resourceAtt,
+                      resource: metricsAtt,
+                    });
```

Comment:
Can we create `createRequestPayload` and `createResponsePayload` which will cover payload generation in whole (same as we do for _metrics_, _traces_ etc.)?

Leaking into this file payload generation doesn't feel right

Reply:
Moved these to the same file we are generating the log payload in 😎 ✅